_content/articles/discourses-on-the-establishments-of-mindfulness-quoted-in-samathadeva's-abhidharmakosopayika-tika_dhammadinna.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Discourses on the establishments of mindfulness (smṛtyupasthānas) quoted in Śamathadeva’s Abhidharmakośopāyikā-ṭīkā"
+authors:
+  - dhammadinna
+external_url: "https://www.academia.edu/37864714/Discourses_on_the_establishments_of_mindfulness_sm%E1%B9%9Btyupasth%C4%81nas_quoted_in_%C5%9Aamathadeva_s_Abhidharmako%C5%9Bop%C4%81yik%C4%81_%E1%B9%AD%C4%ABk%C4%81"
+drive_links:
+  - "https://drive.google.com/file/d/1y528FIdqbrPhmleu6B2XVTtt0rsTi9XC/view?usp=sharing"
+course: sati
+tags:
+  - Smṛtyupasthāna
+  - meditation
+year: 2018
+month: 
+olid: 
+oclc: 
+journal: Journal of Buddhist Studies
+publisher: "Centre for Buddhist Studies, Sri Lanka &
+The Buddha-Dharma Centre of Hong Kong"
+address: "Sri Lanka"
+volume: XV
+pages: "23--38"
+---
+
+A collection of canonical quotations found in the Tibetan Abhidharmakośopāyikā-ṭīkā and their paralles in the Saṃyukta-āgama, all of which highlight the four foundations of mindfulness, their development, arising and their passing away, and the delight in their cultivation .
```

Comment:
Does it include the SA parallels? I think they're just highlighting which SA sutras each passage parallels...